libraries/AP_Networking/Web/scripts/webui.lua

```diff
@@ -0,0 +1,997 @@
+--[[
```

Comment:
why is this added? It seems to be an out of date copy of Tools/AP_Periph/Web/scripts/pppgw_webui.lua

Reply:
removed, will create a separate method to require webui module.